frontend/src/metabase/admin/settings/components/widgets/EmbeddingOption/EmbeddingOption.tsx

```diff
@@ -103,11 +138,11 @@ export const InteractiveEmbeddingOptionCard = () => {
   return (
     <EmbeddingOption
       icon={
-        !isEE || enabled ? (
-          <InteractiveEmbeddingOn />
-        ) : (
-          <InteractiveEmbeddingOff />
-        )
+        <InteractiveEmbedding
```

Comment:
Fix the icon not following custom brand color. Another benefit is we only need to maintain one icon file.